frontend/src/pages/MyAppointmentsPage.jsx

```
@@ -1,10 +1,12 @@
 import React, { useState, useEffect } from "react";
+import { getMyAppointments } from "../api";
 import { getMyAppointments, updateAppointmentStatus } from "../api";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'getMyAppointments'. Consider to delete it or rename it.

'getMyAppointments' is defined here:


(lint/suspicious/noRedeclare)